docs/source/installation/methods.rst

```diff
@@ -0,0 +1,67 @@
+Installation methods
+====================
+
+The following are all of the available options for the installing Ceph through different channels.
+We support 3 main installation methods, all managed by the ``ceph_origin`` variable:
+
+- ``repository``: means that you will get ceph installed through a new repository. Later below choose between 'community', 'rhcs' or 'dev'.
+- ``distro``: means that no separate repo file will be added and you will get whatever version of Ceph is included in your Linux distro.
+- ``local``: means that the ceph binaries will be copied over from the local machine (not well tested, use at your own risk)
+
+Origin: Repository
+--------------------
```

Comment:
These `-` need to be the same length as the title above.

Reply:
I think the rule is *at least* longer than the title, but you're right same length is better :).